gapir/cc/context.cpp

```diff
@@ -123,13 +123,11 @@ bool Context::interpret(bool cleanup, bool isPrewarm) {
     if (api_index == gapir::Vulkan::INDEX) {
       // There is only one vulkan "renderer" so we create it when requested.
       mVulkanRenderer = VulkanRenderer::create();
-      if (mVulkanRenderer->isValid()) {
-        mVulkanRenderer->setListener(this);
-        Api* api = mVulkanRenderer->api();
-        interpreter->setRendererFunctions(api->index(), &api->mFunctions);
-        GAPID_INFO("Bound Vulkan renderer");
-        return true;
-      }
+      mVulkanRenderer->setListener(this);
```

Comment:
What happens if the vulkan renderer is invalid?
Do we want GAPIR to crash in this case?

Reply:
Oh crap. Good catch. Knocked this out by accident!!!